homeassistant/components/acmeda/config_flow.py

```diff
@@ -0,0 +1,122 @@
+"""Config flow for Rollease Acmeda Automate Pulse Hub."""
+import asyncio
+from typing import Dict, Optional
+
+import aiopulse
+import async_timeout
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from .const import DOMAIN, LOGGER  # pylint: disable=unused-import
+from .errors import AuthenticationRequired, CannotConnect
+
+
+class AcmedaFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a Acmeda config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    def __init__(self):
+        """Initialize the config flow."""
+        self.hub: Optional[aiopulse.Hub] = None
+        self.discovered_hubs: Optional[Dict[str, aiopulse.Hub]] = None
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        # This is for backwards compatibility.
+        return await self.async_step_init(user_input)
+
+    async def async_step_init(self, user_input=None):
```

Comment:
```suggestion
    async def async_step_user(self, user_input=None):
```
Please use step_user and delete the above

https://developers.home-assistant.io/docs/config_entries_config_flow_handler/

Reply:
Done.